packages/client/Atmosphere.ts

```diff
@@ -279,11 +273,15 @@ export default class Atmosphere extends Environment {
       }
     }
     const transport = uploadables ? this.baseHTTPTransport : this.transport
-    return transport.fetch({
-      [field]: data,
-      variables,
-      uploadables: uploadables || undefined
-    }, sink || noopSink)
+    return transport.fetch(
+      {
+        [field]: data,
+        variables,
+        uploadables: uploadables || undefined
+      },
+      // if sink is nully, then the server doesn't send a response
+      sink
```

Comment:
graphql-trebuchet-client now handles an optional sink.
if no sink is passed it, the operation isn't cached, which means the server doesn't have to respond